bookdown/02-basics.Rmd

```diff
@@ -512,9 +512,7 @@ This function additionally allows to construct learners with specific hyperparam
 lrn("classif.rpart", id = "rp", cp = 0.001)
 ```
 
-If you pass hyperparameters here, it is added to the default parameters in a `r ref("mlr3misc::insert_named()", text = "insert_named()")`-fashion.
-
-For further information on how to customize learners using mlr3, see the section on [extending learners](#ext-learner).
```

Comment:
Removed this sentence because we referenced `#ext-learner` already above and it looked strange at the end of this paragraph about setting the hyperparameters.